backend/tournesol/resources/export_readme.txt

```diff
@@ -174,3 +181,10 @@ List of columns:
   individual scores. With a small number of contributors, this score would be
   reduced towards 0 such that it is not possible for a small number of users to
   give a very high collective score on a video.
+
+- uncertainty:
+
+  This field is computed by Tournesol's algorithms. It should represent the 
+  uncertainty on the score value. Be careful, due to the score scaling which it
+  may seem very large. For more detail, check out the algorithms implementation 
+  or the paper which describe them https://arxiv.org/abs/2211.01179
```

Comment:
Suggesting to remove the sentence that is not grammatically correct and does not help but confuse more the users. Let's do some analysis on this data and learn better how to document it.

```suggestion
  This field is computed by Tournesol's algorithms. It should represent the 
  uncertainty on the score value. For more detail, check out the algorithms
  implementation or the paper which describe them
  https://arxiv.org/abs/2211.01179
```